### DISCUSSION TITLE
Pygame install

### DISCUSSION THREAD

**Discussion by Laenpo** [community/community#46800] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

pip install pygame
Collecting pygame
  Using cached pygame-2.1.2.tar.gz (10.1 MB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.

When I download pygame on mac it's always like this and I have no idea what's the problem.

**Comment by Hamzalopode** [comment]
Hi, try looking into this: https://www.pygame.org/wiki/MacCompile
Also try using pygame-ce for updated version of pygame.

**Comment by advayc** [comment]
this could be an issue with the version of pip you're installing. Make sure you have the latest version of pip and setuptools installed by running this 

python -m pip install --upgrade pip setuptools

Pygame requires some dependencies to be installed macos. check that you have the necessary dependencies by running this in a terminal window

brew install sdl2 sdl2_gfx sdl2_image sdl2_mixer sdl2_net sdl2_ttf
